Cargo.toml

```diff
@@ -30,13 +30,14 @@ optional = true
 rustix = { version = "0.38", features = ["event"] }
 
 [features]
-default = ["udev", "log", "libinput_1_21"]
+default = ["udev", "log", "libinput_1_26"]
 use_bindgen = ["input-sys/use_bindgen"]
 libinput_1_11 = ["input-sys/libinput_1_11"]
 libinput_1_14 = ["input-sys/libinput_1_14", "libinput_1_11"]
 libinput_1_15 = ["input-sys/libinput_1_15", "libinput_1_14"]
 libinput_1_19 = ["input-sys/libinput_1_19", "libinput_1_15"]
 libinput_1_21 = ["input-sys/libinput_1_21", "libinput_1_19"]
+libinput_1_26 = ["input-sys/libinput_1_26", "libinput_1_21"]
```

Comment:
If this feature was added in 1.23 already, the feature should be named accordingly and the corresponding header file of it's tar-archive should be used for generating.

Reply:
Thanks for pointing this out. Are we implying we should have a 1.23 version here instead of the 1.26 version? I'm not 100% sure what this is pointing out as a feature. Thanks.

Reply:
In general these features are meant to be activated for the minimum-version of libinput users of these bindings want to support. So if you care to enable 1_21 because you need some methods introduced with that version, 1.21 is the minimum libinput version on any target system your crate needs.

Which is why this should be `libinput_1_23`, so that it signifies, that pointer acceleration profiles were added in that version and requiring these options, requires that version of libinput to be available.

Reply:
Removed libinput_1_26 and added libinput_1_23